pom.xml

```diff
@@ -1138,6 +1138,7 @@
               <mixAuditAnalyzerEnabled>false</mixAuditAnalyzerEnabled>
               <nugetconfAnalyzerEnabled>false</nugetconfAnalyzerEnabled>
               <assemblyAnalyzerEnabled>false</assemblyAnalyzerEnabled>
+              <skipSystemScope>true</skipSystemScope>
```

Comment:
I have searched the project. we don't have `<scope>system<scope>` import. Why need this config?

Reply:
It will be failed in jdk8.
https://github.com/apache/bookkeeper/runs/6953541383?check_suite_focus=true